src/ert/shared/models/single_test_run.py

```diff
@@ -14,19 +20,97 @@ def __init__(
         super().__init__(simulation_arguments, ert, local_queue_config, id_)
 
     def checkHaveSufficientRealizations(self, num_successful_realizations: int) -> None:
-        # Should only have one successful realization
+        # Should have at least one successful realization
         if num_successful_realizations == 0:
             raise ErtRunError("Simulation failed!")
 
     def runSimulations(
         self, evaluator_server_config: EvaluatorServerConfig
     ) -> RunContext:
-        return self.runSimulations__(
-            "Running single realisation test ...", evaluator_server_config
+        return asyncio.run(self.run(evaluator_server_config), debug=True)
+        
+    async def run(self, evaluator_server_config: "EvaluatorServerConfig") -> Union[RunContext, None]:
```

Comment:
The `SingleTestRun` inherits from `EnsembleExperiment` instead of `BaseModel`, as it is a simple case of the `EnsembleExperiment` (only single realization, otherwise it is all the same). Hence it uses the same `run` method. I think we can keep this behaviour and avoid duplicating the `run` function. What do you think?